docs/faq/import-beast.md

```diff
@@ -65,5 +65,3 @@ augur import beast --mcc data/beast.mcc.nex --output-tree results/mers.new
     --output-node-data results/beast_data.json
     --most-recent-tip-date 2018.43
 ```
-
-A full [example build can be found here](https://github.com/nextstrain/augur/tree/master/tests/builds/beast_mers).
```

Comment:
This example has since been removed, but I'm guessing it is still useful to reference in the context of this document. If the goal of this PR is to **fix** links, it should not be removed but instead changed to a working link (73b8a71103bae3a36b49acdee7c7c75a7e69a751 is the commit that added this line):

https://github.com/nextstrain/augur/tree/73b8a71103bae3a36b49acdee7c7c75a7e69a751/tests/builds/beast_mers

Reply:
Restored, added a parenthetical about it not being in the active tree.